build-system/compile/single-pass.js

```diff
@@ -559,57 +562,63 @@ exports.singlePassCompile = async function(entryModule, options) {
     .then(wrapMainBinaries)
     .then(intermediateBundleConcat)
     .then(eliminateIntermediateBundles)
-    .then(thirdPartyConcat)
-    .then(removeInvalidSourcemaps)
-    .catch(err => {
-      err.showStack = false; // Useless node_modules stack
-      return Promise.reject(err);
-    });
```

Comment:
Why were these lines removed?

Reply:
The error stacks are useful, they pointed me to the exact line that was failing in this file.

Reply:
It's true they might be useful when you're editing `single-pass.js`, but most of the time, they distract from actual code errors seen while building the runtime. Perhaps you can reinstate the lines here and remove them whenever you're editing this file?

Reply:
Done.